dataunits.rst

```diff
@@ -105,15 +120,15 @@ Python API
 
         Return the DataUnit type with the given name.
 
-    .. py:method:: pack(values)
+    .. py:method:: invariantHash(values)
 
-        Compute an ``bytes`` string that uniquely identifies the given combination of :ref:`DataUnit` values.
+        Compute a hash that is invariant across Python sessions, and hence can be stored in a database.
 
         :param dict values: A dictionary that maps :ref:`DataUnit` type names to either the "values" of those units or actual :ref:`DataUnit` instances.
 
-        :returns: a ``bytes`` object that labels the given combination of units.
+        :returns: a ``bytes`` object that contains the (``sha512``) hash of units.
 
-        This method must be used to populate the ``unit_pack`` field in the :ref:`sql_Dataset` table.
+        This method must be used to populate the ``unit_hash`` field in the :ref:`sql_Dataset` table.
```

Comment:
I think I'd replace the changes to the packing description with a disclaimer that says that we're currently hashing insted (since we think we'll have to return to packing).